agent/connect/ca/provider_aws.go

```diff
@@ -232,7 +232,7 @@ func (a *AWSProvider) ensureCA() error {
 	}
 
 	a.rootPEM = certPEM
-	return nil
+	return nil //
```

Comment:
nit: is this an empty comment?

Reply:
That was definitely accidental. Removing now.